orcid-core/src/main/resources/i18n/messages_en.properties

```diff
@@ -1044,7 +1044,7 @@ orcid.frontend.security.invalid_claim_url=Your claim link seems to be wrong, ple
 orcid.frontend.security.invalid_deactivate_account=You are trying to invalidate an account that doesn't belong to you.
 orcid.frontend.verify.current_work_external_id_invalid=If a value is specified, both the type and id are required
 orcid.frontend.verify.duplicate_email={0} already exists in our system. Would you like to <a href\="/signin">Sign in</a> using {0}?
-orcid.frontend.verify.claimed_email={0} already exists in our system. If this is your email address please contact support to merge the records.
+orcid.frontend.verify.claimed_email={0} already exists in our system. If this is your email address please <a href\="http://orcid.org/help/contact-us" target\="_blank">contact support</a> to merge the records.
```

Comment:
@nitw-shobhit shoot. For additions you don't need to do anything. However for edit you need to run the python script.  Please see the readme at https://github.com/ORCID/ORCID-Source/tree/master/orcid-core/src/main/resources/i18n